CMakeLists.txt

```
@@ -6,7 +6,7 @@
 # the terms of the Apache License 2.0 which accompanies this distribution.     #
 # ============================================================================ #
 
-cmake_minimum_required(VERSION 3.22 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.26 FATAL_ERROR)
```

```suggestion
cmake_minimum_required(VERSION 3.22 FATAL_ERROR)
```
Keeping at 3.22 for now, but this is entirely untested - which is not ideal.